user-guide/Advanced_Modules/User Defined APIs/UD_APIs_Troubleshooting.md

```diff
@@ -71,6 +71,12 @@ The errors in this section can be generated when the UserDefinableApiEndpoint Dx
 
 ## Issues when triggering user-defined APIs
 
+- **Access to XXX at 'XXX' from origin 'XXX' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**
+
+  User-defined APIs don't set CORS headers for safety reasons. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors) for more info.
```

Comment:
Avoid contractions (e.g. "don't") in the user guide. I'd propose to rephrase this as follows:

```suggestion
  For safety reasons, user-defined APIs do not set CORS headers. It is not safe to use a user-defined API from a web client. See [Triggering an API](xref:UD_APIs_Triggering_an_API#cors).
```